components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/valve/TenantContextRewriteValve.java

```diff
@@ -156,6 +160,11 @@ public void invoke(Request request, Response response) throws IOException, Servl
         return rewriteContexts;
     }
 
+    private boolean isTenantQualifiedModeEnabled() {
```

Comment:
Shall we reflect the config from the method name. 
It would be better to use 'isTenantQualifiedUrlsEnabled'